lib/metalsmith.js

```diff
@@ -218,7 +218,10 @@ module.exports = metalsmith(__dirname) // __dirname defined by node.js: name of
       },
       filters: {
         highlight: highlighter,
-        slugger
+        slugger,
+        kebabCase: (string) => {
```

Comment:
We need a new function here since slugger is not designed to turn camelCase strings into kebab-case strings, it's expecting something like a title with spaces between characters.